locales/en/plugin__activemq-artemis-self-provisioning-plugin.json

```diff
@@ -42,29 +42,6 @@
   "Information about {{title}}": "Information about {{title}}",
   "Metrics data is loading": "Metrics data is loading",
   "Data unavailable": "Data unavailable",
-  "Last 5 minutes": "Last 5 minutes",
-  "Last 15 minutes": "Last 15 minutes",
-  "Last 30 minutes": "Last 30 minutes",
-  "Last 1 hour": "Last 1 hour",
-  "Last 6 hours": "Last 6 hours",
-  "Last 12 hours": "Last 12 hours",
-  "Last 1 day": "Last 1 day",
-  "Last 2 days": "Last 2 days",
-  "Last 1 week": "Last 1 week",
-  "Last 2 weeks": "Last 2 weeks",
-  "Refresh Off": "Refresh Off",
-  "15 Seconds": "15 Seconds",
-  "30 Seconds": "30 Seconds",
-  "1 minute": "1 minute",
-  "5 minutes": "5 minutes",
-  "15 minutes": "15 minutes",
-  "30 minutes": "30 minutes",
-  "1 hour": "1 hour",
-  "6 hours": "6 hours",
-  "1 day": "1 day",
-  "2 days": "2 days",
-  "1 week": "1 week",
-  "2 weeks": "2 weeks",
```

Comment:
It's strange that these had to be removed. I think we do still need them, but differently:

"1d": "1 day"

Otherwise the labels aren't good

<img width="678" height="986" alt="Image" src="https://github.com/user-attachments/assets/6113eb00-a650-47fc-8335-eedc70c5f706" />

Reply:
The problem is that `yarn i18n` is going to try to be smart for us and remove these..